doc/sphinx-guides/source/api/native-api.rst

```diff
@@ -2992,9 +2996,37 @@ Deletes an ``AuthenticatedUser`` whose ``id``  is passed. ::
 
     DELETE http://$SERVER/api/admin/authenticatedUsers/id/$id
     
-Note: If the user has performed certain actions such as creating or contributing to a Dataset or downloading a file they cannot be deleted.
+Note: If the user has performed certain actions such as creating or contributing to a Dataset or downloading a file they cannot be deleted. If a user cannot be deleted for this reason, you can choose to :ref:`disable a user<disable-a-user>`.
+    
+.. _disable-a-user:
+
+Disable a User
+~~~~~~~~~~~~~~
+
+Disables an ``AuthenticatedUser`` whose ``identifier`` (without the ``@`` sign) is passed. ::
+
+    PUT http://$SERVER/api/admin/authenticatedUsers/disable/$identifier
     
+Disables an ``AuthenticatedUser`` whose ``id``  is passed. ::
+
+    PUT http://$SERVER/api/admin/authenticatedUsers/disable/id/$id
     
+Note: A primary purpose of most Dataverse installations is to serve an archive. In the archival space, there are best practices around the tracking of data access and the tracking of modifications to data and metadata. In support of these key workflows, a simple mechanism to delete users that have performed edit or access actions in the system is not provided. Providing a Disable User endpoint for users who have taken certain actions in the system alongside a Delete User endpoint to remove users that haven't taken certain actions in the system is by design.
+
+Disabling a user with this endpoint will:
+
+- Disable the user's ability to log in to the Dataverse installation
+- Remove the user's access from all Dataverse collections, datasets and files
+- Prevent a user from being assigned any roles
+- Cancel any pending file access requests generated by the user
+- Remove the user from all groups
+- No longer have notifications generated or sent by the Dataverse installation
```

Comment:
We need to add something that prevents a user from recreating a disabled account with the same username and/or email.

Reply:
@mheppler Can you talk a bit more about why? Is this the case of malicious user or something else?

Reply:
The first bullet here suggests we disable a user account in order to "disable the user's ability to log in to the Dataverse installation"... seems obvious that we don't leave the front door of the house open if we're trying to add a lock to the back door. I don't have a specific use case in mind. I don't see a list of use cases here in the guides, so I can't point to an existing one. Not trying to expand the scope or make more work for anyone. Just making sure we achieve what we set out to do.

Reply:
@mheppler thanks. As a concrete example, you're saying that if I have an account with danny@harvard.edu that gets disabled by an admin, I wouldn't be able to create a new account using danny@harvard.edu?

Reply:
Yeah, email or username, both fields are validated on create to be unique.

Reply:
Just chiming in to make sure I understand. You both seem to be using "builtin" user examples (which is fine) but it applies equally to shib/oauth/oidc. For example, if my ORCID account is disabled on the Dataverse side, I am no longer able to create a Dataverse account using my ORCID account. Instead of a conflict of email addresses (like with builtin users) what's conflicting is the unique identifier within ORCID. Same with Shib (eppn is the unique identifier we use), etc.

Reply:
I pushed https://github.com/IQSS/dataverse/commit/e9845c077ef847e7bb57feb95c67ef7859519389 to reflect my understanding but am not super happy with the phrasing. Feedback welcome.